rca/project_styleguide/templates/patterns/base.html

```diff
@@ -38,38 +38,46 @@
         {% include "patterns/atoms/sprites/sprites.html" %}
 
         {% wagtailuserbar %}
-
         <header class="app__header">
             {% block header %}{% endblock %}
         </header>
 
         <nav class="app__navigation">
-            <section class="header__menus header__menus--desktop">
-                <div class="header__inner--top">
-                    {% secondarynav %}
-                    <form action="{% url 'search' %}" method="get" role="search" class="form form--header-search">
-                        <input class="input" type="text" placeholder="Search…" name="query"{% if search_query %} value="{{ search_query }}{% endif %}">
-                        <button class="button" type="submit">
-                            <svg class="icon icon--magnifying-glass"><use xlink:href="#magnifying-glass"></use></svg>
-                        </button>
-                    </form>
+            <section class="header__menus">
+                <div class="header__inner">
+                    {% include 'patterns/atoms/logo/logo.html' %}
+                    {% include 'patterns/molecules/menu-toggle/menu-toggle.html' with modifier="header__menu-toggle--small" %}
                 </div>
+                <div class="header__inner">
+                    <div class="header__navigation-toggle">
+                        {% include 'patterns/molecules/menu-toggle/menu-toggle.html' with modifier="header__menu-toggle--medium" %}
+                        {% include 'patterns/molecules/search-toggle/search-toggle.html' %}
 
-                {% primarynav %}
-            </section>
+                        <a class="header__menu-close" href="#" data-close-menu>
+                            <svg width="14" height="14"><use xlink:href="#close"></use></svg>
+                        </a>
+                    </div>
+
+                    <div class="nav" data-nav>
+                        <p>Nav here</p>
+                        {% primarynav %}
+                        {% secondarynav %}
+                    </div>
 
-            <section class="header__menus header__menus--mobile" data-mobile-menu>
-                {% primarynav %}
+                    <div class="search" data-search>
+                        <p>Search here</p>
```

Comment:
Meant to be commited :P?

Reply:
left it in for testing by DM however they can probable see the difference in toggle opacity so will remove